src/Service.Tests/dab-config.MsSql.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "https://github.com/Azure/data-api-builder/releases/download/vmajor.minor.patch/dab.draft.schema.json",
   "data-source": {
     "database-type": "mssql",
-    "connection-string": "Server=tcp:127.0.0.1,1433;Persist Security Info=False;User ID=sa;Password=REPLACEME;MultipleActiveResultSets=False;Connection Timeout=5;",
+    "connection-string": "Server=tcp:127.0.0.1,1433;Persist Security Info=False;Trusted_Connection=True;TrustServerCertificate=True;MultipleActiveResultSets=False;Connection Timeout=5;",
```

Comment:
I don't think this change was intended? `TrustServerCertificate=True` may be fine when testing locally, but ideally shouldn't be a setting we encourage in our sample files. This setting will also be overridden in mssqlqyueryexecutor

Reply:
yeah thanks